src/Tribe/Aggregator/Service.php

```diff
@@ -355,11 +355,13 @@ public function has_eventbrite_authorized() {
 		$response = $this->get( 'eventbrite/validate', $args );
 
 		// If we have an WP_Error we return only CSV
-		if ( $response instanceof WP_Error ) {
+		if ( is_wp_error( $response ) ) {
```

Comment:
Why this? `instanceof` is faster and is what the function is doing.

Reply:
I just went with what we used in other places in our code. but you are right. Moving back to `instanceof WP_Error`